File: BuenDoctorAPI/Entities/Appointment.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BuenDoctorAPI.Entities
{
    public partial class Appointment
    {
        public string AppointmentId { get; set; }
        public string PatientId { get; set; }
        public string DataUserId { get; set; }
        public DateTime? Date { get; set; }
        public byte AppointmentStatusId { get; set; }

        public virtual AppointmentStatus AppointmentStatus { get; set; }
        public virtual DataUser DataUser { get; set; }
        public virtual Patient Patient { get; set; }
    }
}

File: buen_doctor_app/lib/models/appointment.dart
import 'package:buen_doctor_app/models/data_user.dart';
import 'package:flutter/material.dart';

import 'appointment_status.dart';
import 'patient.dart';

class Appointment {
  late final String appointmentId;
  late final String patientId;
  late final String dataUserId;
  late final String date;
  late final int appointmentStatusId;

  late final AppointmentStatus appointmentStatus;
  late final DataUser user;
  late final Patient patient;
}
